docs/references/configuration_options_reference.md

```diff
@@ -293,6 +295,10 @@ String used to define the configuration of the broker if one is autogenerated. S
 - `--timeout=` milliseconds to wait for all the federates to connect to the broker (can also be entered as a time like '10s' or '45ms')
 - `--network_timeout=` - Time to establish a socket connection in ms. Times can also be entered as strings such as "15s" or "75ms".
 - `--error_timeout=` - Time in ms to wait after an error state is reached before terminating. Times can also be entered as strings such as "15s" or "75ms".
+- `--query_timeout=` - Time in ms to wait for a query to complete. Times can also be entered as strings such as "15s" or "75ms".
+- `--children=` - The minimum number of child objects the broker should expect before allowing entry to the initializing state.
```

Comment:
I would prefer just one term, probably `subbrokers`.  If you think there will be Intergrators or Modelers who need the more specific term of `children` I could live with that.

Reply:
I needed some term that applied to cores as well as brokers.  Subbrokers doesn't make sense on cores but it is the most obvious on brokers.